src/ensembl/src/content/app/entity-viewer/components/entity-viewer-content/EntityViewerContent.scss

```diff
@@ -0,0 +1,38 @@
+@import 'src/styles/common';
+
+.entityViewerContent {
+  display: grid;
+  background-color: $black;
+  grid-template-columns: 1fr 5fr 2fr;
+  grid-template-rows: auto;
+  padding: 15px;
+  width: 100%;
+}
+
+// NOTE:
+// Remove the background colours and the heights once the PR is approved.
+// They are for demostration purposes only.
+
+.featureImage {
+  background-color: lightgrey;
+  height: 150px;
+  grid-area: 1 / 1 / 2 / 3;
+}
+
+.viewInLinks {
+  background-color: lightblue;
+  height: 150px;
+  grid-area: 1 / 3 / 2 / 4;
+}
+
+.entityViewerTabs {
+  background-color: lightyellow;
+  height: 75px;
```

Comment:
Shouldn't this height be defined entirely by the content of this row? (If so, it's unnecessary)